Zend/tests/enum/final.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Enum is final
+--FILE--
+<?php
+
+enum Foo {}
+
+class Bar extends Foo {}
+
+?>
+--EXPECTF--
+Fatal error: Class Bar may not inherit from final class (Foo) in %s on line %d
```

Comment:
Wouldn't it be better if the error message said `Class Bar may not inherit from an Enum in $s on line %d`?

Reply:
I'm afraid this will open a can of worms, as there are tons of similar error messages...